Cocina/Cocina/Settings/Cells/EditProfileHeader.swift

```diff
@@ -9,9 +9,14 @@ import UIKit
 import SwiftUI
 import SnapKit
 
+protocol EditProfileHeaderDelegate {
```

Comment:
Let's constrain this protocol to `AnyObject` so we can mark the delegate as weak.